app/upw/templates/placement-details/availability.njk

```diff
@@ -56,71 +56,72 @@
       <form method="post" action="{{ action }}">
         <input type="hidden" name="x-csrf-token" value="{{ csrfToken }}">
 
-        <fieldset class="govuk-fieldset" aria-describedby="individual_availability-hint">
-          <legend class="govuk-fieldset__legend individual_availability govuk-label--m">
-            {{ questions['individual_availability'].questionText }}
-          </legend>
-
+        <fieldset class="govuk-fieldset" aria-describedby="individual_availability-legend">
           <table class="govuk-table upw-availability-table">
+            <caption class="govuk-visually-hidden">
```

Comment:
I've added the `caption` here do that the `table` is clearly identifiable when navigating using Voiceover